src/tests/ftest/util/ior_utils.py

```diff
@@ -226,7 +225,8 @@ def get_aggregate_total(self, processes):
 
         return total
 
-    def get_launch_command(self, manager, attach_info, processes, hostfile):
+    def get_launch_command(self, manager, attach_info, processes, hostfile,
+                           client_log):
```

Comment:
To avoid breaking other calls, shouldn't this be optional (`client_log=None`).  Also missing a docstring